File: Assets/common/script/MeshFlipper.js
#pragma strict

function Start () {
	//get a reference to the mesh
	var mesh : Mesh = GetComponent(MeshFilter).mesh;
	//reverse triangle winding
	var triangles:int[]=mesh.triangles;
	var numpolies : int =triangles.Length / 3;
	
	for(var t=0;t<numpolies;t++)
	    {
	    var tribuffer : int = triangles[t*3];
	    triangles[t*3]=triangles[(t*3)+2];
	    triangles[(t*3)+2]=tribuffer;
	    }
	
	mesh.triangles=triangles;
}

function Update () {

}

File: Assets/common/script/MeshFlipper.cs
using UnityEngine;
using System;


public class MeshFlipper:MonoBehaviour{
	
	public void Start() {
		//get a reference to the mesh
		Mesh mesh = GetComponent<MeshFilter>().mesh;
		//reverse triangle winding
		int[] triangles=mesh.triangles;
		int numpolies =triangles.Length / 3;
		
		for(int t=0;t<numpolies;t++)
		    {
		    int tribuffer = triangles[t*3];
		    triangles[t*3]=triangles[(t*3)+2];
		    triangles[(t*3)+2]=tribuffer;
		    }
		
		mesh.triangles=triangles;
	}
	
	public void Update() {
	
	}
}